### PR TITLE
Add notebook demonstrating using BLIP-2 to label a dataset

### PR DESCRIPTION
**Pull request by kfahn22**
This notebook is very similar to the one using OWLv2-- I just changed the model to BLIP-2 and haven't added annotated images.

### CODE REVIEW

File: notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb [hunk 1]
```diff
@@ -0,0 +1,12070 @@
+{
```
Comment (MKhalusova) [inline notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb:1]: <p>which greatly reduces*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/johko/computer-vision-course/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Comment by MKhalusova** [comment]
Great notebook, only a minor nit on my part.

File: notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb (hunk 1 above)
Comment (kfahn22) [reply notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb:1]: <p>Thanks!</p>

Comment (johko) [inline notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #13.    # Set the number of images to generate
</pre><p>Shouldn't it be "Set the number of images to <strong>label</strong>"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/johko/computer-vision-course/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Review by johko — APPROVED** [review]
Just a really minor nit, but else LGTM :+1:

**Review by merveenoyan — APPROVED** [review]
Thank you!

File: notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb (hunk 1 above)
Comment (kfahn22) [reply notebooks/Unit 10 - Synthetic Data Creation/image_labeling_BLIP_2.ipynb:1]: <p>Thanks!</p><p> </p>

**Pull request merged**